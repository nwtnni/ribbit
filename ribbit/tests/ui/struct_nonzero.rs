#[ribbit::pack(size = 32, nonzero)]
struct Bad(u32);

fn main() {}
