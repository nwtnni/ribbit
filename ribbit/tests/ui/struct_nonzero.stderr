error: At least one field must be non-zero for struct to be non-zero
 --> tests/ui/struct_nonzero.rs:1:27
  |
1 | #[ribbit::pack(size = 32, nonzero)]
  |                           ^^^^^^^
